lib/java/os-environment/src/main/java/module-info.java

```diff
@@ -3,6 +3,7 @@
  * Image.
  */
 module org.enso.os.environment {
+  requires org.enso.persistance;
```

Comment:
Also add `requires org.enso.runtime.parser`. You need that one because you are using `Persistables`. There are no failures so far, because we are not running unit tests in JVM.